pipeline_noted.ipynb

```diff
@@ -99,14 +110,18 @@
     "* `minian_path` should be the path that contains a folder named **\"minian\"** , under which the actual codes of **minian** (.py files) resides. The default value **\\.** means \"current folder\", which should work in most cases unless you want to try out another specific version of minian that is not in the same folder as this notebook.\n",
     "* `dpath` is the folder that contains the actual videos, which are usually named **\"msCam\\*.avi\"** where \\* is a number.\n",
     "* `chunks` is used to divide data into smaller chunks for parallel processing. Currently it is rarelly used and has little impact on the actual performance of the script. It is recommended to leave `chunks` to default. \n",
-    "* Both `subset` and `subset_mc` are used to subset the data. `subset` operates on the whole data. A good usecase for this is to take out some troublesome frames or take out some bad pixels, for example, those used as timestamps in wireless miniscope recording. `subset_mc` operates only on the input data to motion correction, and thus only affect how the shifts are calculated. A good usecase for this is if you have good landmark (e.g blood vessels) in the field of view and want to `estimate_shifts` only on those region of interest. Note that if you are running in interactive mode (`interactive = True`), there is an interactive viewer later to help you find proper `subset_mc`. Both `subset` and `subset_mc` should be specified as python [dictionary](https://docs.python.org/3/tutorial/datastructures.html#dictionaries), whose `keys` are the dimension along which subsetting should be done, and `values` specify how subsetting should be done (usually a [slice object](https://docs.python.org/3/c-api/slice.html)). If the word \"dimension\" and how data are indexed are confusing, move on to the section where the data structure are discussed.\n",
     "* `in_memory` specify whether to load the whole dataset into memory.\n",
-    "* `interactive` controls whether interactive plots should be shown. Not that plotting the results implies actual computation to be carried out, and thus could be very inefficient when the data are not in the memory. In real situation you would probably want `in_memory = True` as well if your data can fit in the memory.\n",
-    "* `output_size` controls the realtive size of all the plots on a scale of 0-100 percent, though it can goes beyond 100 without any problem. Adjust this to please your eye."
+    "* `interactive` controls whether interactive plots should be shown. In real situation, when you want to visualize interactive plots while figuring out the best parameters, you would want `in_memory=True` and `interactive=True` as long as your data can fit in the memory. On the other hand, once you finalized your parameters and ready for batch processing, you would want both of them to be set as False.\n",
```

Comment:
I still want the user to understand why those two parameters usually go together and what happens if they don't